docs/_pages/rtm_client.md

```diff
@@ -165,24 +170,28 @@ web.channels.list()
 
 ### Listening for message subtypes
 
-There's a shortcut syntax for listening to specific message subtypes, just format the event type as
+There's a shortcut syntax for listening to specific [message
+subtypes](https://api.slack.com/events/message#message_subtypes), just format the event type as
 `message::${message_subtype}`. See the example below.
 
 ```javascript
+const { RTMClient } = require('@slack/client');
+// SNIP: the initialization code shown above is skipped for brevity
```

Comment:
This confused me more than helped (especially the word `SNIP`). I feel like you can just remove this line and the line above and just show how the `message::channel_name` event is used

Reply:
this is interesting feedback. we added those lines into almost every example in `web_client.md` based on feedback that it wasn't already clear which lines were required to make the code in any given example work. this was sort of stiking a compromise by just showing the imports at a minimum, but not showing the instantiation because it can become visually "heavy".

i was adding this here because `rtm_client.md` didn't get the same love of retreatment. but now i'm curious if its helpful at all, and if we should make all of the examples shorter. 🤔